src/app/components/profile/pages/profile/profile.component.html

```diff
@@ -50,15 +58,62 @@ <h1>{{ user.userName }}</h1>
           </div>
         </div>
       </div>
+
+      <!-- Auth token (Disabled in third person) -->
+      <div id="annotations-link" class="row mt-3" *ngIf="!thirdPerson">
+        <div class="col-md-12 pb-3">
+          <div class="card">
+            <!-- Header -->
+            <div class="card-header">Authentication Token</div>
+
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item p-1">
+                <small>
+                  <p class="px-1" [innerText]="authTokenDescription"></p>
```

Comment:
Why not
```suggestion
                  <p class="px-1">{{ authTokenDescription }}</p>
```

Reply:
I am growing a preference for `innerText` as its more clear that no HTML will be inserted, and it doesn't insert random whitespace (which makes my tests easier). Is there a reason i should use `{{ }}` over it?

Reply:
I'd say `{{ }}` is the canonical form for adding things to the template.

It should be preferred over binding to a property wherever possible.

Angular has guidance here: https://angular.io/guide/property-binding#property-binding-and-interpolation

> Use either form when rendering data values as strings, though interpolation is preferable for readability. However, when setting an element property to a non-string data value, you must use property binding.

Speculation: binding to innerText (a property will get somewhere) seems more complex and less declarative.

Test helpers should be insensitive to whitespace as well?